packages/cozy-stack-client/src/NotesCollection.spec.js

```diff
@@ -0,0 +1,50 @@
+import NotesCollection from './NotesCollection'
+import CozyStackClient from './CozyStackClient'
+
+describe('NotesCollection', () => {
+  const stackClient = new CozyStackClient()
+  const collection = new NotesCollection(stackClient)
+
+  afterEach(() => {
+    jest.resetAllMocks()
+  })
+
+  afterAll(() => {
+    jest.restoreAllMocks()
+  })
+
+  describe('all', () => {
+    it('should call the appropriate route', async () => {
+      jest.spyOn(stackClient, 'fetchJSON').mockResolvedValue({
+        data: [],
+        links: {},
+        meta: { count: 0 }
+      })
+
+      await collection.all()
+      expect(stackClient.fetchJSON).toHaveBeenCalledWith('GET', '/notes')
+    })
```

Comment:
Normalization is not tested, is it expected  ?

Reply:
No, will add it.